jobs/env/ci-kubernetes-e2e-scalability-highspeed-common.env

```diff
@@ -0,0 +1,4 @@
+# Increase throughput in master components.
+SCHEDULER_TEST_ARGS=--kube-api-qps=100 --kube-api-burst=100
+CONTROLLER_MANAGER_TEST_ARGS=--kube-api-qps=100 --kube-api-burst=100
+APISERVER_TEST_ARGS=--max-requests-inflight=3000 --max-mutating-requests-inflight=1000
```

Comment:
This will make it 3000, 1000 even for 2k-node clusters (where currently we're using 1500, 500). Do we want that?

Reply:
I took a look into what we're doing upstream, and it seems we are setting (1500, 500) by default for large enough clusters:
https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/gci/configure-helper.sh#L1459

So I removed it from here, and left 3000, 1000 only for 5k node cluster.

Since this is needed, I think we should change upstream to use those values too and then got rid of it completely.

Reply:
I agree.